core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -420,6 +420,14 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val STORAGE_REMOTE_SHUFFLE_MAX_DISK =
+    ConfigBuilder("spark.storage.remote.shuffle.maxDisk")
+      .doc("Maximum disk space to use to store shuffle blocks before rejecting remote " +
```

Comment:
Could you add some description about what happens when we reject the remote shuffle blocks?

Reply:
Sure, good idea :)